### PR TITLE
DICOMWeb fixes for Google hosted service

### PR DESCRIPTION
**Pull request by PaulHax**
- Remove trailing slash if on host URL
- Support minimal metadata responses
- Add a `dicomweb=http://localhost:5173/dicom-web` URL parameter to configure default host

### CODE REVIEW

File: src/store/dicom-web/dicom-web-store.ts [hunk 1]
```diff
@@ -43,10 +45,17 @@ async function getAllPatients(host: string): Promise<PatientInfo[]> {
...
-    ? ref(process.env.VUE_APP_DICOM_WEB_URL)
-    : useLocalStorage<string>('dicomWebHost', '');
-  const isConfigured = computed(() => !!host.value);
+  const urlParams = vtkURLExtract.extractURLParameters() as UrlParams;
+  const dicomWebFromURLParam = urlParams.dicomweb as string | undefined;
+  const initialHost = dicomWebFromURLParam ?? process.env.VUE_APP_DICOM_WEB_URL;
+
+  const host = initialHost
+    ? ref(initialHost)
+    : useLocalStorage<string | null>('dicomWebHost', ''); // null if cleared by vuetify text input
```
Comment (floryst) [inline src/store/dicom-web/dicom-web-store.ts:54]: Should `initialHost` also be saved to local storage, or do you think the `dicomweb` parameter should be an overriding exception?

File: src/store/dicom-web/dicom-web-store.ts [hunk 2]
```diff
@@ -43,10 +45,17 @@ async function getAllPatients(host: string): Promise<PatientInfo[]> {
  * Collect DICOM data from DICOMWeb
  */
 export const useDicomWebStore = defineStore('dicom-web', () => {
-  const host = process.env.VUE_APP_DICOM_WEB_URL
-    ? ref(process.env.VUE_APP_DICOM_WEB_URL)
-    : useLocalStorage<string>('dicomWebHost', '');
-  const isConfigured = computed(() => !!host.value);
+  const urlParams = vtkURLExtract.extractURLParameters() as UrlParams;
+  const dicomWebFromURLParam = urlParams.dicomweb as string | undefined;
```
Comment (floryst) [inline src/store/dicom-web/dicom-web-store.ts:49]: I'd recommend moving `dicomweb` into a constant. It makes it more obvious if we need to ever configure it down the road.

Comment (PaulHax) [reply src/store/dicom-web/dicom-web-store.ts]: Done.

**Comment by floryst** [comment]
LGTM!

**Pull request merged**